src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -109,18 +118,34 @@ def _migrate_views(self):
             self.index(force_refresh=True)
         return all_tasks
 
-    def _migrate_table(self, src_table: TableToMigrate, mounts: list[Mount], hiveserde_in_place_migrate: bool = False):
+    def _migrate_table(
+        self,
+        src_table: TableToMigrate,
+        mounts: list[Mount],
+        hiveserde_in_place_migrate: bool = False,
+        managed_table_external_storage: str | None = None,
```

Comment:
it can't be `str | None`, it must be `str`. a default has to be provided on the level of `migrate_tables` method, where you convert value to uppercase. all private methods must take `str`, not `.. | None`.